code/game/objects/items/extinguisher.dm

```diff
@@ -267,8 +271,14 @@
 
 		user.visible_message(span_notice("[user] empties out \the [src] onto the floor using the release valve."), span_info("You quietly empty out \the [src] using its release valve."))
 
-//firebot assembly
+//firebot assembly and reagent filling
 /obj/item/extinguisher/attackby(obj/O, mob/user, params)
+	if(istype(O, /obj/item/reagent_containers))
+		if(safety)
+			to_chat(user, "<span class='warning'>You need to take off the safety before you can refill the [src]!</span>")
```

Comment:
use `span_warning` instead of sending HTML
`to_chat(user, span_warning("You need to take off the safety before you can refill the [src]!"))`